install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#TODO(by black7375)
+#However, there may be existing user's settings, so back up and make a copy.
+#I just made a simple one yesterday, assuming that there may be multiple backup states.
+#I think it's better to do a redirection(>>) after backup to merge the contents of the file.
+printf "Are you using ESR, Default, or Dev?(ESR/Default/Dev): "
+read -r
+if [ "ESR" == "$REPLY" ] || [ "$REPLY" == "esr" ]; then
+  profiledir="$(grep '.default-esr' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
```

Comment:
Use functions for repetitive code.

Reply:
Using functions could be tricky in treating directories and could cause unexpected results. I will fix this later on as an optimization.